ui/src/util/auth/provider.tsx

```diff
@@ -41,7 +41,13 @@ export const AuthProvider: React.FC<{ children: React.ReactNode }> = ({
       if (!token) throw new Error("No token received");
       setToken(token);
       localStorage.setItem("jwt", token);
-      window.postMessage({ command: "parakeet-setAuthToken", token }, origin);
+
+      const refreshToken = res.data.status.data.refresh_token;
```

Comment:
this nested value retrieval will throw an error if the shape of the response data ever changes. I'm not sure if it will make anything more visible in terms of the chrome extension logs, but I would consider wrapping lines 45-50 in a try / catch:

```js
try {
  // all this stuff
} catch (e) {
  console.error('Error obtaining refresh token', e)
}
```